sql-cli/sql_cli/__main__.py

```diff
@@ -251,11 +251,17 @@ def init(
         help=f"(Optional) Set the DAGs Folder. Default: {DEFAULT_DAGS_FOLDER}",
         show_default=False,
     ),
+    data_dir: Path = typer.Option(
+        None,
+        dir_okay=True,
+        help=f"(Optional) Set the data directory. Default: {DEFAULT_DATA_DIR}",
```

Comment:
Can you please add more information here about what a "data directory" is? I might not immediately know if I'm just playing around with the CLI (and we can never fully depend on people to read the docs ;) )

Reply:
Thanks. Yes, will do! 👍